include/template/gcc-defs.h

```diff
@@ -31,6 +31,7 @@
 #define _Alignof(x) (sizeof(x) & 0xf)
 #define __attribute__(x)
 #define __builtin_add_overflow(x,y,z) ((x), (y), (z), 1)
+#define __builtin_add_overflow_p(x,y,z) ((x), (y), (z))
```

Comment:
It's better to use 1 as the result. The value of z isn't used for any other reason than check its type.

Reply:
Done